src/zc/buildout/buildout.py

```diff
@@ -384,7 +384,12 @@ def __init__(self, config_file, cloptions,
 
         download_cache = options.get('download-cache')
         extends_cache = options.get('extends-cache')
+
+        if bool_option(options, 'abi-tag-eggs', 'false'):
+            from zc.buildout.pep425tags import get_abi_tag
+            options['eggs-directory'] += '-' + get_abi_tag()
```

Comment:
FWIW, if possible I would prefer to have the api-specific eggs-directory *beneath* the egg directory I specify (or is default). This prevents cluttering of the top-level directory. It also means that a bunch of .gitignore files won't have to be updated to ignore a bunch of new 'eggs-pypy', etc, directories.

In my case, I have one main eggs-directory set in ~/.buildout/default.cfg. I have a different backup policy set for that directory. Some people may place that directory on larger (slower) storage. That directory may also have different permissions than its containing directory. It would be nice not to have to make those changes for more directories (whose name you can only guess at until they're created).